src/docfx/build/legacy/metadata/LegacyMetadata.cs

```diff
@@ -40,13 +40,20 @@ public static JObject GenerataCommonMetadata(JObject metadata, Docset docset)
             return newMetadata;
         }
 
+        public static JObject GenerateLegacyRedirectionRawMetadata(Docset docset, PageModel pageModel)
+            => new JObject
+            {
+                ["redirect_url"] = pageModel.RedirectionUrl,
+                ["locale"] = docset.Config.Locale,
```

Comment:
>["locale"] = docset.Config.Locale [](start = 16, length = 33)

why we need `locale`

Reply:
for publishing usage

---
In reply to: [210468903](https://github.com/dotnet/docfx/pull/3208#discussion_r210468903) [](ancestors = 210468903)